src/20241127_Multi_FluidAlignment/FluidAlignment_20241127.s.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers, IPayloadsControllerCore, PayloadsControllerUtils} from 'aave-helpers/src/GovV3Helpers.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {EthereumScript, ArbitrumScript, BaseScript} from 'solidity-utils/contracts/utils/ScriptUtils.sol';
+import {AaveV3Ethereum_FluidAlignment_20241127} from './AaveV3Ethereum_FluidAlignment_20241127.sol';
+import {AaveV3EthereumLido_FluidAlignment_20241127} from './AaveV3EthereumLido_FluidAlignment_20241127.sol';
+import {AaveV3Arbitrum_FluidAlignment_20241127} from './AaveV3Arbitrum_FluidAlignment_20241127.sol';
+import {AaveV3Base_FluidAlignment_20241127} from './AaveV3Base_FluidAlignment_20241127.sol';
+
+/**
+ * @dev Deploy Ethereum
+ * deploy-command: make deploy-ledger contract=src/20241127_Multi_FluidAlignment/FluidAlignment_20241127.s.sol:DeployEthereum chain=mainnet
+ * verify-command: FOUNDRY_PROFILE=mainnet npx catapulta-verify -b broadcast/FluidAlignment_20241127.s.sol/1/run-latest.json
+ */
+contract DeployEthereum is EthereumScript {
+  function run() external broadcast {
+    // deploy payloads
+    address payload0 = GovV3Helpers.deployDeterministic(
+      type(AaveV3Ethereum_FluidAlignment_20241127).creationCode
+    );
+    address payload1 = GovV3Helpers.deployDeterministic(
+      type(AaveV3EthereumLido_FluidAlignment_20241127).creationCode
+    );
+
```

Comment:
Missing deploy script for `AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127` payload contract.
Also the `AaveV3Ethereum_FluidAlignment_TokenTransfers_20241127` payload should be added on the proposal creation script too